Dockerfile

```diff
@@ -19,9 +20,12 @@ RUN apk add --no-cache \
                 make \
                 python
 
-RUN wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://raw.githubusercontent.com/sgerrand/alpine-pkg-glibc/master/sgerrand.rsa.pub
-RUN wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.27-r0/glibc-2.27-r0.apk
-RUN apk add glibc-2.27-r0.apk
+#
+ENV ALPINE_PKG_GLIBC_VERSION 2.28-r0
+
+RUN wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub
```

Comment:
sgerrand.com tls certificate has expired. Any other alternatives?

Reply:
So the certificate for https://alpine-pkgs.sgerrand.com is ok, but not ok for sgerrand.com

Reply:
Checked https://alpine-pkgs.sgerrand.com which is ok. Pinged developer. Lets keep it for now.